app/queries/list_movies_query.rb

```diff
@@ -28,7 +28,7 @@ def prepare_collection
   end
 
   def search_movies
-    @results = @results.where('title ILIKE ?', "%#{query}%")
+    @results = @results.where('title LIKE ?', "%#{query}%")
```

Comment:
I will add some
```ruby
'title LIKE LOWER(?)', "%#{query}%"
```
Or
```ruby
'title LIKE ?', "%#{query.downcase}%"
```
To be sure that we find what we want even if someone pass: `inception` or `Inception` or `InCePTioN`